mlflow/genai/judges/prompts/completeness.py

```diff
@@ -0,0 +1,20 @@
+# NB: User-facing name for the completeness assessment.
+COMPLETENESS_ASSESSMENT_NAME = "completeness"
+
+COMPLETENESS_PROMPT = """\
+Consider the following user prompt and AI response.
```

Comment:
nit: let's be consistent in terminology (e.g., AI vs assistant) - my preference is `assistant`